openhcl/openhcl_boot/src/main.rs

```diff
@@ -287,18 +288,20 @@ fn shim_parameters(shim_params_raw_offset: isize) -> ShimParams {
 }
 
 /// The maximum number of reserved memory ranges that we might use.
-///
-/// 1. VTL2 parameter regions (could be up to 2).
-/// 2. Sidecar image.
-/// 3. One reserved range per sidecar node.
-pub const MAX_RESERVED_MEM_RANGES: usize = 3 + sidecar_defs::MAX_NODES;
+/// See ReservedMemoryType definition for details.
+pub const MAX_RESERVED_MEM_RANGES: usize = 5 + sidecar_defs::MAX_NODES;
 
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 enum ReservedMemoryType {
+    /// VTL2 parameter regions (could be up to 2).
     Vtl2Config,
     Vtl2Reserved,
+    /// Sidecar image.
     SidecarImage,
+    /// A reserved range per sidecar node.
     SidecarNode,
+    /// Preserved DMA buffers and hardware queues.
+    DmaBuffers,
```

Comment:
ditto with renaming this, because it's really a persisted GPA pool or whatever.

Reply:
Renamed